spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -678,6 +655,202 @@ boolean result = pollableSource.poll(received -> {
+	capability of the underlying messaging middleware.
+
+	- *application* - the error handling is done within the application (custom error handler)
+
+Spring Cloud Stream uses https://github.com/spring-projects/spring-retry[Spring Retry] library to facilitate successful message processing. See <<Retry Template>> for more details.
+However, when all fails the exceptions thrown by the message handlers are propagated back to the binder.  At that point binder invokes custom error handler or communicates
+the error back to the messaging system (re-queue, DLQ etc).
+
+===== Application Error Handling
+
+Let’s first look at the application level error handling.
+
+For each input binding, Spring Cloud Stream creates a dedicated error channel with the following semantics `<channel-name>.<group-name>.errors`.
+If such channel has a subscriber all errors will be sent to that subscriber.
+
+Consider the following:
+
+[source,text]
+----
+spring.cloud.stream.bindings.input.group=myGroup
+----
+
+[source,java]
+----
+@StreamListener(Sink.INPUT) // destination name 'input.myGroup'
+public void handle(Person value)
+	throw new RuntimeException("BOOM!");
+}
+
+@StreamListener(Processor.INPUT + ".myGroup.errors") //channel name 'input.myGroup.errors'
+public void error(Message<?> message) {
+	System.out.println("Handling ERROR: " + message);
+}
+----
+
+The `handle(..)` method which subscribes to the channel named `input` throws an exception.  Given there is also a subscriber to the error channel `input.myGroup.errors`
+all error messages will be handled by this subscriber.
+
+I the event you have multiple bindings you may want to chose to have a single error handler. Spring Cloud Stream automatically provides support for
+_global error channel_ by bridging each individual error channel to the channel named `errorChannel`, allowing a single subscriber to handle all errors.
+
+[source,java]
+----
+@StreamListener(“errorChannel")
+public void error(Message<?> message) {
+	System.out.println("Handling ERROR: " + message);
+}
+----
+
+This may be a convenient option if error handling logic is the same regardless of which handler produced the error.
+
+Also, error messages sent to the `errorChannel` can be published to a specific destination at the broker by configuring a binding named `error` for the outbound target.
+This option provides a mechanism to automatically send error messages to another application bound to that destination or for later retrieval (e.g., audit).
+For example, to publish error messages to a broker destination named `myErrors`, set the following property:
+
+[source,text]
+----
+spring.cloud.stream.bindings.error.destination=myErrors.
+----
+
+===== System Error Handling
+
+System level error handling implies that the errors are communicated back to the messaging system and given that not every messaging system
```

Comment:
Replace with "System-level error handling implies that the errors are communicated back to the messaging system and, given that not every messaging system"